roles/ceph-common/defaults/main.yml

```diff
@@ -63,6 +63,13 @@ ceph_origin: 'upstream' # or 'distro'
 #
 ceph_use_distro_backports: false # DEBIAN ONLY
 
+ceph_mon_packages:
+  - ceph
```

Comment:
I'm confused, you mentioned that the `ceph` package is actually installing all the Ceph packages.
So shouldn't we only have the `ceph-mon`package?
Am I missing something?